rmgpy/data/kinetics/library.py

```diff
@@ -245,6 +245,7 @@ def getLibraryReactions(self):
                                       duplicate=entry.item.duplicate, reversible=entry.item.reversible,
                                       allow_pdep_route=entry.item.allow_pdep_route,
                                       elementary_high_p=entry.item.elementary_high_p)
+                rxn.family = self.label #the library the reaction was loaded from (opposed to originally from)
```

Comment:
This works because we also check the family attribute before adding a reaction as "duplicate"?

Reply:
When we check for existing reactions, we use the family attribute to identify the "source" of a reaction, regardless of whether it's a LibraryReaction or TemplateReaction. The library attribute is not used for this check. When saving Chemkin files, we do use the library attribute to write the source library in the comments.

So this fix does something which was not originally intended, but happens to do exactly what we want, by using the family attribute to store the actual source (the seed library) and the library attribute to store the original source (the original library).